src/c_api.rs

```diff
@@ -86,7 +86,13 @@ pub extern "C" fn JSONAPI_openKey<'a>(
         let bytes = slice::from_raw_parts(bytes as *const u8, len);
         String::from_utf8_lossy(bytes).into_owned()
     };
-    json_api_open_key_internal(RedisJsonKeyManager, ctx, &key) as *mut c_void
+    json_api_open_key_internal(
+        RedisJsonKeyManager {
+            phantom: std::marker::PhantomData,
```

Comment:
Adding a use declaration would shorten the code (occurs many times)
from `std::marker::PhantomData` to `PhantomData`

Reply:
Fixed